vite.config.js

```diff
@@ -24,14 +24,14 @@ export default defineConfig(({ mode }) => {
       host: "0.0.0.0",
       proxy: {
         "/rest": {
-          target: "http://localhost:10880/",
+          target: `http://localhost:${env.VITE_BACKEND_PORT ?? '8080'}/`,
```

Comment:
The improvement is nice, but avoid changing configuration on unrelated merge requests